templates/base.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Taxi Service</title>
+    {% load static %}
+    <link rel="stylesheet", href="{% static 'css/styles.css' %}">
+</head>
+<body>
+    {% block sidebar %}
+        {% include "includes/sidebar.html"%}
+    {% endblock %}
+
+    {% block content %}
+    {% endblock %}

```

Comment:
use `container-fluid` to split the page in blocks and got better view

Reply:
![image](https://user-images.githubusercontent.com/108726137/192499184-8252fca2-fe87-45b0-8690-e25af42c8c79.png)